src/wp-content/themes/twentyeleven/readme.txt

```diff
@@ -23,7 +23,7 @@ For more information about Twenty Eleven please go to https://codex.wordpress.or
 
 == Copyright ==
 
-Twenty Eleven WordPress Theme, Copyright 2011-2024 WordPress.org & Automattic.com
+Twenty Eleven WordPress Theme, Copyright 2011-2024 WordPress.org, Automattic.com and contributors.
```

Comment:
@sabernhardt Good questions, I'm checking how best to credit Automattic. 

WordPress.org and the WordPress Foundation are different entities (the former belonging to Matt, the latter a non-profit) so I think that can remain as is.

Reply:
This is a great flag @sabernhardt. The Foundation and w.org are separate entities (if a website can be called an entity!), but I'm not sure about how Automattic prefers to be referred to. I will ask and return with what answers I can.

Reply:
I have returned with confirmation!

1. The WordPress ones should stay as they are
2. Automattic.com should be changed to "Automattic Inc." [sic]

Reply:
@sabernhardt I left the order as is as I don't know if it makes a difference. Thanks again for asking the Automattic question, it didn't even begin to occur to me.

Reply:
The order probably should remain as it is, but part of my reasoning was to avoid "Automattic Inc. and contributors." (we use a period with abbreviations like that in the U.S.). The serial comma could help with that instead.
"WordPress.org, Automattic Inc., and contributors."